src/test/Components/CreateImageWizard/CreateImageWizard.test.js

```diff
@@ -692,7 +672,7 @@ describe('Step Review', () => {
             name: 'Amazon Web Services'
         });
         userEvent.click(buttonRegistration);
-        screen.getByText('Register the system on first boot');
+        screen.getByText('Register the system later');
```

Comment:
This does disable testing the subscription tab on the review page when adding one, no?

Reply:
I'm not completely sure what your question is. I tried to switch the default selection for registration to no registration since it is simpler to test and for the tests that aren't directly checking registration I think it is better for it to be simpler. The subscription tab displays regardless of what is selected in the step and we test the registration key in the `should allow choosing activation keys` test

Reply:
Oh I missed line 800 too, it thought the first boot option wasn't tested now, sorry for the confusion